chart/templates/registry-facade-daemonset.yaml

```diff
@@ -22,6 +22,8 @@ spec:
       kind: pod
       stage: {{ .Values.installation.stage }}
       gitpod.io/nodeService: registry-facade
+  updateStrategy:
+    type: OnDelete
```

Comment:
onDelete seems to be default behaviour, since we force rollingUpdate on kubectl, will this override ? is that the thought ?
https://cloud.google.com/kubernetes-engine/docs/how-to/updating-apps#considerations_for_statefulsets_and_daemonsets

Reply:
OnDelete was default in earlier versions. Default is now RollingUpdate